methods.ts

```diff
@@ -8,6 +8,7 @@ export interface MethodInfo {
   loop?: boolean;
   special?: 'txt'|'bp';
   c?: number;
+  sub?: string;
```

Comment:
Can you also add this to `scripts/geninstr.ts`, line 383.

Reply:
Done. Can you document how to use this script?

Reply:
It's kind of complicated. You need to extract `instructions.lua` from the game's main mod, then run it through extractor.cjs to convert it to json. However I think I also had to hand patch luaparse to handle lua 5.4 syntax